modules/Bio/EnsEMBL/Compara/PipeConfig/Metazoa/ProteinTrees_conf.pm

```diff
@@ -48,36 +48,28 @@ sub default_options {
         'division'   => 'metazoa',
 
         # homology_dnds parameters:
-        'taxlevels' => ['Drosophila' ,'Hymenoptera', 'Nematoda'],
+        'taxlevels' => ['Eukaryota'],
 
         # GOC parameters:
-        'goc_taxlevels' => ['Diptera', 'Hymenoptera', 'Nematoda'],
+        'goc_taxlevels' => ['Eukaryota'],
 
         # HighConfidenceOrthologs parameters:
         # In this structure, the "thresholds" are for resp. the GOC score, the WGA coverage and %identity
+
         'threshold_levels' => [
             {
-                'taxa'          => [ 'Euteleostomi', 'Ciona' ],
-                'thresholds'    => [ 50, 50, 25 ],
-            },
-            {
-                'taxa'          => [ 'Aculeata', 'Anophelinae', 'Caenorhabditis', 'Drosophila', 'Glossinidae', 'Onchocercidae' ],
+                'taxa'          => [ 'Penaeus', 'Thoracicalcarea', 'Daphnia', 'Apinae', 'Haliotis', 'Echinacea', 'Asteroidea', 'Anthozoa', 'Scleractinia' ],
```

Comment:
https://github.com/Ensembl/ensembl-compara/blob/42c698bcd17f95efa1e4d59aff99a21a1adf10e3/conf/metazoa/mlss_conf.xml#L96C13-L96C13

Would it be a good idea to add Parachela to this group  to account for the two tardigrade species ? 
https://www.ncbi.nlm.nih.gov/Taxonomy/Browser/wwwtax.cgi?mode=Info&id=149990&lvl=3&lin=f&keep=1&srchmode=1&unlock